orchagent/dash/dashvnetorch.cpp

```diff
@@ -331,11 +331,21 @@ bool DashVnetOrch::addVnetMap(const string& key, VnetMapBulkContext& ctxt)
     bool exists = (vnet_map_table_.find(key) != vnet_map_table_.end());
     if (!exists)
     {
-        addOutboundCaToPa(key, ctxt);
-        addPaValidation(key, ctxt);
+        bool vnet_exists = (gVnetNameToId.find(ctxt.vnet_name) != gVnetNameToId.end());
+        if (vnet_exists)
+        {
+            addOutboundCaToPa(key, ctxt);
+            addPaValidation(key, ctxt);
+        }
+        else
+        {
+            SWSS_LOG_INFO("Not creating VNET map for %s since VNET %s doesn't exist", key.c_str(), ctxt.vnet_name.c_str());
+        }
+        return false;
```

Comment:
this should be moved to `else case`

Reply:
Resolved offline, keep this here so the `consumer` is not modified for post-bulk operations.